src/main/java/com/blackducksoftware/integration/hub/api/component/ComponentRestService.java

```diff
@@ -48,4 +50,20 @@ public ComponentRestService(final RestConnection restConnection, final Gson gson
         return allComponents;
     }
 
+    public ComponentItem getExactComponentMatch(String id, String groupId, String artifactId, String version)
```

Comment:
one note here. The 'id' parameter, its not an actual id. It's the forge or namespace of the ID. A better name for this parameter would probably be forge (namespace is probably ok too).